### ISSUE TITLE
Fix use of long in conch for Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8508]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8508|
|Type|enhancement|
|Created|2016-06-27 08:17:24Z|
|Branch|https://github.com/twisted/twisted/tree/8508-rodrigc-conch|

A few places in conch use **L** for long literals.  This is invalid in Python 3, and needs to be fixed.

<details><summary>Searchable metadata</summary>

```
trac-id__8508 8508
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8508_rodrigc_conch 8508-rodrigc-conch
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467015444576075 1467015444576075
changetime__1467047831272631 1467047831272631
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/231

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Thanks! Please merge.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset a548c420dd46be60d15df0f5afc153d3c522481a
```
#!CommitTicketReference repository="" revision="a548c420dd46be60d15df0f5afc153d3c522481a"
Merge 8508-rodrigc-conch: Fix use of long in conch for Python 3

Author: rodrigc
Reviewer: adiroiban
Fixes: #8508
```
